examples/flutter_gallery/lib/demo/playground/home.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'playground_demo.dart';
+
+class PlaygroundScaffold extends StatelessWidget {
+  const PlaygroundScaffold({
```

Comment:
Although it's not necessary here, it's still best to include a `Key, key` parameter and pass it along with `: super(key: key)` for the sake of copy and pasters.